algorithms/src/snark/marlin/data_structures/proof.rs

```diff
@@ -448,7 +451,7 @@ mod test {
             for j in 1..11 {
                 let test_with_none = i * j % 2 == 0;
                 let commitments = rand_commitments(i, j, test_with_none);
-                let batch_sizes = vec![i; j];
+                let batch_sizes = vec![i as u32; j];
```

Comment:
nit: this (perfectly valid) cast could be avoided by iterating over `1u32..11`; applies to all such cases